smartsnippets_inherit/static/admin/smartsnippet_inherit/js/plugininherit_form.js

```diff
@@ -0,0 +1,205 @@
+(function($) {
+
+    _editor = {
+        _initial: {},
+        _variablesSubmission: false,
+        getInfoBox: function(text, type){
+            return "<span class='ssvar-info " + (type? type: '') + "'>" + text + "</span>"
+        },
+        getVariablesEl: function(){
+            return $(this.options.variables)
+                .find("[name^='_'][name$='_'][id^='var_']");
+        },
+        getVariableLabel: function(varEl){
+            var varsBox = $(this.options.variables);
+            return (
+                varsBox.find('label[for="' + varEl.attr('id') + '"]').text() ||
+                varEl.attr('name').replace(/(^_)|(_$)/g, '')
+            );
+        },
+        variablesToString: function(varsEl){
+            var self = this;
+            return $.map(varsEl, function(el){
+                return self.getVariableLabel($(el));
+            }).join(', ');
+        },
+        registerInitialData: function(snippetId){
+            var self = this;
+            self._initial[snippetId] = {};
+            self.getVariablesEl().each(function(){
+                self._initial[snippetId][$(this).attr('name')] = $(this).val();
+            });
+        },
+        submitOverwriteData: function(snippetId){
+            var self = this;
+            return function() {
+                var varsBox = $(self.options.variables);
+                if (!SnippetWidgetRegistry.allValid()){
+                    return false;
+                }
+                var post_data = {'snippet_plugin': snippetId};
+                var varsEl = [];
+                $.each(self._initial[snippetId], function(name, initial_value){
+                    var varEL = $('[name="' + name + '"]');
+                    var current_value = varEL.val();
+                    if (initial_value != current_value){
+                        varsEl.push(varEL);
+                        post_data[name] = current_value;
+                    }
+                });
+                if (varsEl.length === 0){
+                    alert('No variables changed');
+                    return ;
+                }
+                if(!confirm("The following variables will be overwritten: " +
```

Comment:
it's cleaner to extract the confirm box answer in a variable then check it's value like you did on line 82